client/components/language-picker/modal.jsx

```diff
@@ -417,7 +415,33 @@ export class LanguagePickerModal extends PureComponent {
 		this.handleClose();
 	};
 
-	handleClose = () => {
+	recordLanguagePickedEvent = ( isClosingWithoutSelection ) => {
```

Comment:
Noting that the name here is a bit confusing: it implies that we're recording an event when we "pick" a language, but instead what we actually do is we record an event when a language is selected AND changed. Should we make that more obvious in the function name?